webpack.common.js

```diff
@@ -0,0 +1,40 @@
+const path = require("path");
```

Comment:
Webpack recommends this pattern of having `common`, `dev`, and `prod` files and merging the needed configurations together.